src/concuerror.erl

```diff
@@ -26,7 +26,7 @@ start(Options, LogMsgs) ->
   _ = [?log(Logger, Level, Format, Args) || {Level, Format, Args} <- LogMsgs],
   SchedulerOptions = [{logger, Logger}|LoggerOptions],
   {Pid, Ref} =
-    spawn_monitor(concuerror_scheduler, run, [SchedulerOptions]),
+    spawn_monitor(fun() -> concuerror_planner:run(SchedulerOptions) end),
```

Comment:
The reason I recently changes `spawn_monitor/1` with `spawn_monitor/3` here is that it makes debugging (e.g. via the `debugger` or `observer` app) a little easier. Any reason for "reverting" this?

Reply:
Not really, also there are some record statements repeating at concuerror_planner. I repeated themmm instead of putting themm at concuerror.hrl because I didn't know if you wanted them visible at the scheduler or not.